packages/styleguides/docs/src/05.flavors.md

```diff
@@ -0,0 +1,101 @@
+* EU-flavor components are placed into `./src/flavors/eu`.
+* Visual regression testing configuration accept an optional `FLAVOR` environment variable. For further details, please check visual regression testing documentation pages.
+
+### Development notes
+
+The fact that ECL is to accommodate 2 separate sets of style guides, for each flavor, means that each flavor-specific style guide needs to be a separate fractal instance. Being a separate fractal instance bring these benefits:
+
+* Concrete modifications in one flavor-specific style guide visualization will not affect the other style guide.
+* Build process is straight-forward, an [default build capabilities of fractal](https://fractal.build/guide/project-settings) are used
+* Work on components from a given flavor is independent from the work on components of the other flavor
+
+At the same time, splitting work in per-flavor focus also means there are constraints:
+
+* Templates of components in one flavor cannot be used by templates of the other flavor, i.e. `fractal.components.set` configuration option.
+
+In short, this means that each flavor will have an independent set of:
+
+* project configurations for title, statuses, etc. - anything defined in `fractal.js`
+* components' templates
+* `ecl-builder.config.js` configurations about how CSS, JS and other assets are managed
+* each project lives and progresses independently from the other
+
+### Maintaining flavors (Important notes on APIs)
+
+The split of the project introduces several important rules which should be followed in order to keep ECL project repository maintainable for contributors, and also easy to integrate by consumers of ECL builds and resources.
+
+#### Naming
+
+Each flavor should be named in a distinguishable manner:
+
+* In `package.json` one would have `@ec-europa/eu-logos`. `@ec-europa` namespace should be preserved for correct organization on [npmjs](https://www.npmjs.com/). The `eu-logos` is for the EU flavor, whereas `ec-logos` should be the one for the EC flavor.
+* Components' files are to be named with `eu-` or `ec-` based on flavors: `eu-logos.twig`, `eu-logos.scss`, `eu-logos.config.js`, etc. This makes flavor-specific code to be easily discoverable in search, clear and simple enough, following existing patterns. (no underscores and suffixes)
+* Twig templates should be prefixed with `@ec-europa/` in order to be consistently consumable by [Europa Component Library Twig loader](https://github.com/openeuropa/ecl-twig-loader)
+
+Please note that all current EC-flavor components have preserved `@ec-europa/ecl-*` naming in order to give time for consumers of ECL components to plan changes based on flavors. The `@ec-europa/ecl-` namespace, in future, should be the place to store reusable parts of components such as mixins and variables.
+
+Also, for keeping current consumption of ECL releases smooth for users, releases are to include `framework.zip` with this structure:
```

Comment:
Missing `framework` root folder, on purpose? When I unzip an archived named `framework.zip`, my first expectation is to get a `framework` folder (currently the case)

Reply:
I will remove this part, no worries